app/javascript/spotlight/admin/blocks/pages_block.js

```diff
@@ -7,20 +7,15 @@ SirTrevor.Blocks.FeaturedPages = (function(){
 
     icon_name: "pages",
 
-    autocomplete_url: function() { return $(this.inner).closest('form[data-autocomplete-exhibit-pages-path]').data('autocomplete-exhibit-pages-path').replace("%25QUERY", "%QUERY"); },
+    autocomplete_url: function() { return document.getElementById(this.instanceID).closest('form[data-autocomplete-exhibit-pages-path]').dataset.autocompleteExhibitPagesPath; },
+    autocomplete_fetch: function(url) {
+      return this.fetchOnceAndFilterLocalResults(url);
+    },
     autocomplete_template: function(obj) {
       const thumbnail = obj.thumbnail_image_url ? `<div class="document-thumbnail"><img class="img-thumbnail" src="${obj.thumbnail_image_url}" /></div>` : ''
       return `<div class="autocomplete-item${!obj.published ? ' blacklight-private' : ''}">${thumbnail}
-      <span class="autocomplete-title">${obj.title}</span><br/><small>&nbsp;&nbsp;${obj.description}</small></div>`
+      <span class="autocomplete-title">${this.highlight(obj.title)}</span><br/><small>&nbsp;&nbsp;${obj.description}</small></div>`
```

Comment:
Is there a description for pages? This is displaying `undefined` in the UI for me:
<img width="1138" alt="Screenshot 2024-09-16 at 10 08 04 AM" src="https://github.com/user-attachments/assets/1b16ea34-9db3-49ba-a30c-563a2c7a01c9">

Reply:
Thanks for catching that. That's the current behavior (at least in the test app with the fixtures) but I have been changing those as I find them in this PR.

I don't _think_ there are descriptions for pages. Do we want to pick something else for inclusion in this work?